Sources/Confidence/Confidence.swift

```diff
@@ -329,6 +350,13 @@ extension Confidence {
         }
 
         public func build() -> Confidence {
+            var debugLogger: DebugLogger?
+            if loggerLevel != LoggerLevel.NONE {
+                debugLogger = DebugLoggerImpl()
+                debugLogger?.logContext(context: initialContext)
+            } else {
+                debugLogger = nil
+            }
```

Comment:
I think it would be good (and not really unorthodox) to have warnings logged even if the user doesn't supply a `LoggerLevel`.

Reply:
But in that case we still instantiate a Logger, which doesn't really make sense if you don't want a logger?

Reply:
ok.. maybe we shouldn't instantiate when passing `NONE`, but if possible we should then default to `WARN` and log everything that the SDK deems to be a warning (and higher).